packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+@mixin bone-width($width) {
+  &.pt-skeleton-bone-#{$width} {
+    width: $width * 1%;
+  }
+}
+
+.pt-skeleton-bone {
+  border-radius: 2px;
+  background-color: rgba($gray1, 0.2);
+  width: 100%;
+  height: $pt-grid-size * 0.6;
+
+  // negative animation delay ensures animations are synced
+  animation: ($pt-transition-duration * 20) linear ($pt-transition-duration * -70) infinite glow;
+
+  @include bone-width(100);
+  @include bone-width(95);
+  @include bone-width(90);
+  @include bone-width(85);
+
+  & + & {
+    margin-top: $pt-grid-size;
+  }
+}
+
+.pt-skeleton-content {
+  opacity: 0;
+  animation: ($pt-transition-duration * 5) ease-in forwards fade-in;
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes glow {
+  0% {
+    background-color: rgba($gray4, 0.2);
+  }
+
+  50% {
+    background-color: rgba($gray1, 0.2);
+  }
+
+  100% {
+    background-color: rgba($gray4, 0.2);
+  }
+}
+
+/*
+Skeleton
+
+`Skeleton`s allow you to conditionally show a loading animation in places where you have content that fits into rectangular
```

Comment:
There is no more formal `Skeleton` anymore (it's not a react component), so just refer to it as a normal noun (perhaps italicized in its first usage).

In addition, you should outline the overall API we are providing here and how it must be customized to fit users' needs.